### PR TITLE
chore: hide edgeless debug menu entry

### PR DESCRIPTION
**Pull request by doodlewind**

### CODE REVIEW

**Review by Himself65 — APPROVED** [review]
Breaking changes on e2e test

File: packages/playground/src/components/debug-menu.ts [hunk 1]
```diff
@@ -410,7 +409,9 @@ export class DebugMenu extends LitElement {
 
         <div
           class="edgeless-toolbar"
-          style=${'display:' + (this.mode === 'edgeless' ? 'flex' : 'none')}
+          style=${
+            'display:none' /*'display:' + (this.mode === 'edgeless' ? 'none' : 'none')*/
```
Comment (himself65) [inline packages/playground/src/components/debug-menu.ts:413]: remove?

Comment (doodlewind) [reply packages/playground/src/components/debug-menu.ts:413]: Well I'd rather keep it here till the new self-contained toolbar is landed.

/cc @thorseraq

Comment (thorseraq) [reply packages/playground/src/components/debug-menu.ts:413]: Thanks, current debug menu is a good reference

**Pull request merged**